src/main/java/ca/bc/gov/chefs/etl/constant/PCDConstants.java

```diff
@@ -339,521 +314,225 @@ public class PCDConstants extends Constants {
-		});
-		HEADERS.put(DECISION_LOG_COMMENTS, new String[] {
-				"SUBMISSION_ID", "COMMENT_ID", "COMMENT", "COMMENT_DATE"
-		});
-		HEADERS.put(DECISION_LOG_INITIATIVES, new String[] {
-				"SUBMISSION_ID", "INITIATIVE_NAME", "INITIATIVE_TYPE"
-		});
+		HEADERS.put(DECISION_LOG_SUBMISSIONS,
+				new String[] {"SUBMISSION_ID", "CREATED_AT", "LATE_ENTRY", "SUBMITTER_FULL_NAME", "SUBMITTER_USER_NAME", "SUBMITTER_EMAIL", "SUBMISSION_STATUS", "SUBMISSION_VERSION",
+						"SUBMISSION_FORM_NAME", "HEALTH_AUTHORITY", "COMMUNITY_NAME", "TYPE_OF_INITIATIVE", "REQUEST_NUMBER", "SUBMITTED_DATE", "DESCRIPTION", "REQUEST_CATEGORY", "REQUEST_STATUS",
+						"RECOMMENDED_DOCUMENTATION_TYPE", "OTHER_DOCUMENTS", "DATE_DECISION_MADE", "COMMENTS_RECOMMENDATIONS", "FINAL_DECISION_COMMENTS", "DECISION_MADE_BY", "FINAL_DECISION",
+						"FINAL_DOCUMENTS_RECEIVED", "PRECEDENT_SETTING", "UPDATED_APPROVAL_TRACKER", "UPDATED_FINANCIAL_REPORT", "NOT_ALL_PCNS", "RSN_FOR_EXC_IN_BDGT_CHG_DT",
+						"SUGGEST_DUE_DT_DECISION", "FISCAL_YEAR", "RECEIVED_BY_DSB_FINANCE", "BUDGET_CHANGE", "BUDGET_CHANGE_DT"});
+		HEADERS.put(DECISION_LOG_CHANGE_REQUEST_FILE_UPLOAD, new String[] {"ID", "URL", "SIZE", "STORAGE", "ORIGINAL_NAME", "SUBMISSION_ID"});
+		HEADERS.put(DECISION_LOG_PCN_NAMES, new String[] {"SUBMISSION_ID", "PCN_NAME", "TYPE"});
+		HEADERS.put(DECISION_LOG_COMMENTS, new String[] {"SUBMISSION_ID", "COMMENT_ID", "COMMENT", "COMMENT_DATE"});
+		HEADERS.put(DECISION_LOG_INITIATIVES, new String[] {"SUBMISSION_ID", "INITIATIVE_NAME", "INITIATIVE_TYPE"});
 
 		/**
 		 * HA Hierarchy
 		 */
-		HEADERS.put(HA_HIERARCHY_HEALTH_AUTHORITY, new String[] {
-				"SUBMISSION_ID", "CREATED_AT", "LATE_ENTRY", "SUBMITTER_FULL_NAME", "SUBMITTER_USER_NAME",
-				"SUBMITTER_EMAIL", "SUBMISSION_STATUS", "SUBMISSION_VERSION", "SUBMISSION_FORM_NAME", "HEALTH_AUTHORITY"
-		});
-		HEADERS.put(HA_HIERARCHY_COMMUNITY, new String[] {
-				"SUBMISSION_ID", "COMMUNITY_ID", "COMMUNITY_NAME", "HSIAR_SERVICE_PLAN_GAP_ANALYSIS",
-				"HSIAR_SERV_PL_GAP_ANLYS_DATE"
-		});
-		HEADERS.put(HA_HIERARCHY_PRIMARY_CARE_NETWORK, new String[] {
-				"COMMUNITY_ID", "PRIMARY_CARE_NETWORK_ID", "PCN_NAME", "PCN_TYPE", });
-		HEADERS.put(HA_HIERARCHY_PRIMARY_CARE_INITIATIVES, new String[] {
-				"PRIMARY_CARE_NETWORK_ID", "PRIMARY_CARE_INITIATIVE_ID", "INITIATIVE_NAME", "INITIATIVE_TYPE",
-				"TYPE_OF_CARE"
-		});
-		HEADERS.put(HA_HIERARCHY_CLINIC, new String[] {
-				"PRIMARY_CARE_NETWORK_ID", "CLINIC_ID", "CLINIC_NAME", "CLINIC_TYPE"
-		});
+		HEADERS.put(HA_HIERARCHY_HEALTH_AUTHORITY, new String[] {"SUBMISSION_ID", "CREATED_AT", "LATE_ENTRY", "SUBMITTER_FULL_NAME", "SUBMITTER_USER_NAME", "SUBMITTER_EMAIL", "SUBMISSION_STATUS",
+				"SUBMISSION_VERSION", "SUBMISSION_FORM_NAME", "HEALTH_AUTHORITY", "HEALTH_AUTHORITY_CODE"});
+		HEADERS.put(HA_HIERARCHY_COMMUNITY, new String[] {"SUBMISSION_ID", "COMMUNITY_ID", "COMMUNITY_NAME", "HSIAR_SERVICE_PLAN_GAP_ANALYSIS", "HSIAR_SERV_PL_GAP_ANLYS_DATE", "PCN_COMMUNITY_CODE"});
```

Comment:
I'm following up to see if it's too late to change the column name but I prefer COMMUNITY_CODE.

Reply:
This rename should be resolved in the latest commit!